src/main.js

```diff
@@ -0,0 +1,180 @@
+// Mobile menu functionality
+import {
+  menuIconMobile, menuLinks, openMenu, closeMenu,
+} from './menu.js';
+
+// Popup window functionality
+import { createModal, projects } from './popup.js';
+
+//  Validation
+import {
+  isItUppercase, email, form, error,
+} from './validation.js';
+
+// Preserve data
+import { storageAvailable } from './localstorage.js';
+
+// Start Mobile menu functionality ------------------------------------->
+menuIconMobile.addEventListener('click', openMenu);
+menuLinks.forEach((menuLink) => {
+  menuLink.addEventListener('click', closeMenu);
+});
+// FInish Mobile menu functionality ------------------------------------->
+
+// Start Popup window functionality ------------------------------------->
+document.addEventListener('DOMContentLoaded', () => {
+  createModal();
+
+  const buttonsProject = document.querySelectorAll('.works__card button');
+  const closeIcon = document.querySelector('.closeModal');
+  const modal = document.querySelector('.modal');
+  const backgroundModal = document.querySelector('.overlay');
+  const projectName = document.querySelector('.project-name');
+  const projectFeature = document.querySelector('.project-feature');
+  const projectBack = document.querySelector('.project-background');
+  const projectYear = document.querySelector('.project-year');
+  const projectImage = document.querySelector('.project-image');
+  const projectDescription = document.querySelector('.project-description');
+  const projectTool1 = document.querySelector('.project-tool1');
+  const projectTool2 = document.querySelector('.project-tool2');
+  const projectTool3 = document.querySelector('.project-tool3');
+  const projectLinkLive = document.querySelector('.project-linkLive');
+  const projectLinkSource = document.querySelector('.project-linkSource');
+
+  const closeModal = () => {
+    modal.classList.remove('active');
+    backgroundModal.classList.remove('active');
+  };
+
+  buttonsProject.forEach((button) => {
+    button.addEventListener('click', () => {
+      if (button.id === '0') {
+        projectName.textContent = `${projects[button.id].name}`;
+        projectFeature.textContent = `${projects[button.id].feature}`;
+        projectBack.textContent = `${projects[button.id].background}`;
+        projectYear.textContent = `${projects[button.id].year}`;
+        projectImage.setAttribute('src', `${projects[button.id].image}`);
+        projectDescription.textContent = `${projects[button.id].description}`;
```

Comment:
- [ ] I think I will go with the approach suggested by the previous reviewer, it is easy and straightforward to use a `for loop`, iterate over the projects array, and render the `project[i].type`, please make sure you correct the code as for now it is not clean, remember, we are all learner if you are not informed of anything, it is our must to help you use the correct approach.